README.md

```diff
@@ -7,6 +7,12 @@
 
 A Julia interface for running a test FTP server with [pyftpdlib](https://pyftpdlib.readthedocs.io/en/latest/index.html).
 
+## Prerequisites
+
+FTPServer makes use of PyCall for running the FTP server.
+It's recommended that you use [`PYTHON=""`](https://github.com/JuliaPy/PyCall.jl#specifying-the-python-version) so that Julia uses it's own Python distribution.
```

Comment:
```suggestion
It's recommended that you use [`PYTHON=""`](https://github.com/JuliaPy/PyCall.jl#specifying-the-python-version) so that Julia uses its own Python distribution.
```